openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -323,23 +323,7 @@ def add_staff_markup(user, disable_staff_debug_info, block, view, frag, context)
         return frag
     # TODO: make this more general, eg use an XModule attribute instead
     if isinstance(block, VerticalBlock) and (not context or not context.get('child_of_vertical', False)):
-        # check that the course is a mongo backed Studio course before doing work
-        is_studio_course = block.course_edit_method == "Studio"
```

Comment:
I'm not 100% on the details here, but where is the equivalent of this check in the new code?  Seems like in the old code we don't show the link unless the `course_edit_method` is "Studio", but I don't see a similar condition in the new stuff.

Reply:
Hrmmm, good point...

We're also not performing this check in the MFE; we always link to Studio.

@davidjoy With this implemented as-is, I don't think we can perform this check; we don't have access to the `block`, so I think we'd need another approach.

@marcotuts (cc: @ormsbee)  Do you have background on this feature, its use/exposure/etc?

Depending on that, we should determine:

1. If we care about this check, do we need to port this check to the MFE too? (now or backlog)
2. If we don't, do we care enough to preserve backward compatibility here? (now or backlog)
3. Or just ignore it and always link to the block in Studio?